documentation/docs/pages/contributing/Add-Supported-Config.md

```diff
@@ -0,0 +1,59 @@
+
+### Scope
+There are ACCESS-OM3 global and regional configurations (ACCESS-OM3 and ACCESS-rOM3) that are under active development at ACCESS-NRI and within the communities we collaborate with. Some of the configurations developed outside of ACCESS-NRI have a wider interest and it will be beneficial for these configurations to be supported and maintained at ACCESS-NRI. Here we provide protocol and guidance on how to apply to have your configuration supported by ACCESS-NRI. 
+
+### What are the benefits?
+When a configuration is supported by ACCESS-NRI then it can be kept up to date with the latest versions of ACCESS-OM3's components, including applying bug fixes, adding new features and upgrades. It will also give more visibility to your configuration, allow others to readily run the configuration and allows for greater community input and collaboration. In some cases, a supported configuration can also benefit from being an official "release" meaning there can be additional documentation and providence of how the configuration was created and run.
+
+### Criteria
+Including community developed configurations will require ongoing upkeep and we cannot support all configurations. Supported configurations will need to minimise the upkeep overhead whilst meeting a community need. The following criteria will be considered when deciding which configurations to support. There can be some flexibility in these criteria for the right configuration so if your configuration doesn't fit all criteria then we encourage you to discuss this with us. Conversely, we may decline configurations that fit the criteria if we are already managing many configurations. We encourage starting a conversation with us early in the development of the configuration so we can plan and assist in meeting the criteria. 
+
+#### Minimising overheads
+The configuration files will need to closely match an existing ACCESS-OM3 or ACCESS-rOM3 configuration to minimise maintenance burden. In particular, the configuration needs to be:
+
+ - running on the NUOPC coupler; 
+ - on Gadi;
+ - runs using `payu`; 
+ - Except for the `MOM_input`, `MOM_override` and `config.yaml` files, the expectation is that files will only differ from an existing configuration by one or two lines. 
+ 
+The `MOM_input` can be very different in  configurations due to the need to specify different parameter choices but the layout and order of these specifications should match layout and order of the ACCESS-OM3 configurations. The `MOM_override` file can be used to add further configuration options such boundary condition specifications. The `config.yaml` file will differ due to the need to specify different input files and executables but the layout of this file should closely match an ACCESS-OM3 configuration, including pointing to an ACCESS-NRI managed executable. This may sound challenging but the development team is here to help contributors understand and meet these criteria upon review.
```

Comment:
To me, the way this is written implies that boundary condition specifications have to be done through `MOM_override`, which isn't true.

```suggestion
The `MOM_input` can be very different in  configurations due to the need to specify different parameter choices but the layout and order of these specifications should match layout and order of the ACCESS-OM3 configurations. The `config.yaml` file will differ due to the need to specify different input files and executables but the layout of this file should closely match an ACCESS-OM3 configuration, including pointing to an ACCESS-NRI managed executable. This may sound challenging but the development team is here to help contributors understand and meet these criteria upon review.
```